x-pack/plugins/observability_solution/observability/public/pages/alert_details/alert_details.tsx

```diff
@@ -68,6 +69,16 @@ const RELATED_ALERTS_TAB_ID = 'related_alerts';
 const ALERT_DETAILS_TAB_URL_STORAGE_KEY = 'tabId';
 type TabId = typeof OVERVIEW_TAB_ID | typeof METADATA_TAB_ID | typeof RELATED_ALERTS_TAB_ID;
 
+const getPageTitle = (ruleCategory: string) => {
```

Comment:
Would you please add [these removed tests](https://github.com/elastic/kibana/pull/193958/files#diff-e4042c05d534cc80f8d90b537e04bce171b8631187b7ae2ad91ecd484e12d854L37) for this function?